build.gradle

```diff
@@ -21,28 +21,27 @@ import java.text.SimpleDateFormat
 
 buildscript {
   repositories {
-    mavenCentral()
     maven { url "https://artifacts.consensys.net/public/maven/maven/" }
   }
   dependencies {
     classpath 'tech.pegasys.internal.license.reporter:license-reporter:1.0.1'
-    classpath 'org.owasp:dependency-check-gradle:10.0.2'
   }
 }
 
 plugins {
+  id 'org.owasp.dependencycheck' version "10.0.3"
   id 'java-test-fixtures'
-  id 'com.diffplug.spotless' version '6.2.0'
-  id 'com.github.ben-manes.versions' version '0.27.0'
-  id 'com.github.jk1.dependency-license-report' version '2.0'
-  id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-  id 'me.champeau.gradle.jmh' version '0.5.3' apply false
-  id 'net.ltgt.errorprone' version '2.0.2'
-  id 'org.ajoberstar.grgit' version '5.2.0'
+  id 'com.diffplug.spotless' version '7.0.0.BETA1'
```

Comment:
Do need to use a beta version of spotless? it's not in production code so it's probably ok.

Reply:
I was double minded on this one, and finally added it to catch any surprising stuff in advance. Since it is build-only as you mentioned, we should be alright about it.